monetio/models/_camx_mm.py

```diff
@@ -448,15 +448,18 @@ def _calc_midlayer_height_agl(dset):
     else:
         raise "No height variable found, but _calc_midlayer_height_agl was called."
     mid_layer_height = np.array(dset[height])  # height in the layer upper interface of each layer
-    layer_height = dset[height]
+    layer_height_agl = dset[height]
+    layer_height_agl.attrs["long_name"] = "layer_height_agl"
+    layer_height_agl.attrs["var_desc"] = "Layer height above ground level at top"
```

Comment:
The calculation suggests to me that this is all the interface heights (nlayers + 1), not just the tops?

Reply:
It's not, because in CAMx height is included as a variable per layer, which only has the altitude at the top.
the height of the surface is in a different variable (called topo or TOPO_M), which is even in a different file